src/ModalHeader.js

```diff
@@ -13,7 +13,7 @@ class ModalHeader extends React.Component {
         { this.props.closeButton &&
           <button
             className='close'
-            aria-label={this.props['aria-label'] || 'Close'} //eslint-disable-line react/prop-types
+            aria-label={this.props['aria-label'] || 'Close'}
             onClick={this.props.onHide}
```

Comment:
Ha-ha :smile: 
The elefant in the room wasn't noticed by me at all :blush: 
(My mind was tuned only to `eslint` comments)

Thank you @jquense :+1: